[2023-05-04T01:50:47.000+0000] {processor.py:153} INFO - Started process (PID=34) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:50:47.005+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:50:47.006+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:50:47.006+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:50:48.617+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:50:48.607+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GitHubCloneOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T01:50:48.617+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:50:48.675+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.687 seconds
[2023-05-04T01:51:19.015+0000] {processor.py:153} INFO - Started process (PID=48) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:51:19.018+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:51:19.020+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:51:19.019+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:51:19.828+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:51:19.826+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GitHubCloneOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T01:51:19.828+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:51:19.849+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.850 seconds
[2023-05-04T01:51:50.169+0000] {processor.py:153} INFO - Started process (PID=61) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:51:50.171+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:51:50.172+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:51:50.172+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:51:51.002+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:51:51.000+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GitHubCloneOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T01:51:51.002+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:51:51.025+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.861 seconds
[2023-05-04T01:52:21.329+0000] {processor.py:153} INFO - Started process (PID=74) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:52:21.333+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:52:21.335+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:52:21.335+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:52:22.115+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:52:22.114+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GitHubCloneOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T01:52:22.116+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:52:22.138+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.818 seconds
[2023-05-04T01:52:31.300+0000] {processor.py:153} INFO - Started process (PID=80) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:52:31.303+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:52:31.312+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:52:31.312+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:52:32.121+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:52:32.119+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T01:52:32.121+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:52:32.142+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.852 seconds
[2023-05-04T01:52:38.230+0000] {processor.py:153} INFO - Started process (PID=91) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:52:38.231+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:52:38.232+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:52:38.232+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:52:38.911+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:52:38.909+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T01:52:38.912+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:52:38.932+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.708 seconds
[2023-05-04T01:53:09.244+0000] {processor.py:153} INFO - Started process (PID=105) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:09.247+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:53:09.248+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:53:09.248+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:09.911+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:53:09.910+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T01:53:09.911+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:09.933+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.694 seconds
[2023-05-04T01:53:13.067+0000] {processor.py:153} INFO - Started process (PID=109) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:13.073+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:53:13.076+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:53:13.075+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:13.745+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:53:13.744+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T01:53:13.746+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:13.767+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.729 seconds
[2023-05-04T01:53:18.436+0000] {processor.py:153} INFO - Started process (PID=113) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:18.438+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:53:18.439+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:53:18.439+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:19.354+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:19.422+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:53:19.421+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:53:19.466+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:53:19.465+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:53:19.497+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.068 seconds
[2023-05-04T01:53:27.364+0000] {processor.py:153} INFO - Started process (PID=119) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:27.372+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:53:27.378+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:53:27.377+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:28.209+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:28.214+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:53:28.213+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:53:28.248+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:53:28.248+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:53:28.275+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.938 seconds
[2023-05-04T01:53:58.440+0000] {processor.py:153} INFO - Started process (PID=132) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:58.442+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:53:58.443+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:53:58.443+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:59.114+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:53:59.197+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:53:59.197+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:53:59.226+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:53:59.225+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:53:59.252+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.821 seconds
[2023-05-04T01:54:30.114+0000] {processor.py:153} INFO - Started process (PID=145) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:54:30.115+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:54:30.115+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:54:30.115+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:54:30.747+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:54:30.769+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:54:30.768+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:54:30.810+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:54:30.810+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:54:30.841+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.732 seconds
[2023-05-04T01:55:01.090+0000] {processor.py:153} INFO - Started process (PID=157) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:55:01.091+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:55:01.091+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:55:01.091+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:55:01.710+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:55:01.729+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:55:01.729+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:55:01.770+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:55:01.770+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:55:01.796+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.711 seconds
[2023-05-04T01:55:32.133+0000] {processor.py:153} INFO - Started process (PID=170) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:55:32.135+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:55:32.136+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:55:32.136+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:55:32.946+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:55:32.967+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:55:32.967+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:55:33.000+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:55:33.000+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:55:33.025+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.896 seconds
[2023-05-04T01:56:03.388+0000] {processor.py:153} INFO - Started process (PID=190) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:56:03.392+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:56:03.393+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:56:03.393+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:56:04.017+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:56:04.036+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:56:04.036+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:56:04.069+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:56:04.069+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:56:04.108+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.734 seconds
[2023-05-04T01:56:34.417+0000] {processor.py:153} INFO - Started process (PID=203) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:56:34.422+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:56:34.426+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:56:34.425+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:56:35.120+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:56:35.144+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:56:35.143+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:56:35.176+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:56:35.176+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:56:35.202+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.807 seconds
[2023-05-04T01:57:05.550+0000] {processor.py:153} INFO - Started process (PID=215) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:57:05.552+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:57:05.553+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:57:05.553+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:57:06.284+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:57:06.305+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:57:06.305+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:57:06.339+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:57:06.339+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:57:06.366+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.827 seconds
[2023-05-04T01:57:36.551+0000] {processor.py:153} INFO - Started process (PID=228) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:57:36.557+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:57:36.562+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:57:36.560+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:57:37.262+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:57:37.283+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:57:37.282+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:57:37.316+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:57:37.315+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:57:37.339+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.813 seconds
[2023-05-04T01:58:07.455+0000] {processor.py:153} INFO - Started process (PID=241) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:58:07.459+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:58:07.460+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:58:07.460+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:58:08.098+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:58:08.119+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:58:08.118+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:58:08.153+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:58:08.153+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:58:08.176+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.730 seconds
[2023-05-04T01:58:38.264+0000] {processor.py:153} INFO - Started process (PID=254) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:58:38.268+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:58:38.270+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:58:38.270+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:58:38.953+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:58:38.973+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:58:38.972+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:58:39.005+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:58:39.005+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:58:39.030+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.777 seconds
[2023-05-04T01:59:09.698+0000] {processor.py:153} INFO - Started process (PID=266) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:59:09.701+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:59:09.702+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:59:09.702+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:59:10.299+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:59:10.318+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:59:10.318+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:59:10.348+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:59:10.348+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:59:10.369+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.681 seconds
[2023-05-04T01:59:40.630+0000] {processor.py:153} INFO - Started process (PID=279) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:59:40.631+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T01:59:40.632+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:59:40.631+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:59:41.312+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T01:59:41.335+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:59:41.334+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T01:59:41.371+0000] {logging_mixin.py:137} INFO - [2023-05-04T01:59:41.371+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T01:59:41.393+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.768 seconds
[2023-05-04T02:00:11.683+0000] {processor.py:153} INFO - Started process (PID=292) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:00:11.685+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:00:11.686+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:00:11.686+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:00:12.427+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:00:12.450+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:00:12.450+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:00:12.485+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:00:12.485+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:00:12.519+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.843 seconds
[2023-05-04T02:00:42.811+0000] {processor.py:153} INFO - Started process (PID=305) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:00:42.812+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:00:42.814+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:00:42.814+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:00:43.443+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:00:43.467+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:00:43.467+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:00:43.504+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:00:43.504+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:00:43.530+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.736 seconds
[2023-05-04T02:00:51.715+0000] {processor.py:153} INFO - Started process (PID=309) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:00:51.716+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:00:51.717+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:00:51.717+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:00:52.339+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:00:52.336+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:00:52.339+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:00:52.369+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.660 seconds
[2023-05-04T02:01:22.687+0000] {processor.py:153} INFO - Started process (PID=321) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:01:22.689+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:01:22.690+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:01:22.690+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:01:23.296+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:01:23.294+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:01:23.296+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:01:23.322+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.644 seconds
[2023-05-04T02:01:53.607+0000] {processor.py:153} INFO - Started process (PID=334) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:01:53.610+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:01:53.611+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:01:53.610+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:01:54.208+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:01:54.206+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:01:54.208+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:01:54.229+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.628 seconds
[2023-05-04T02:02:24.446+0000] {processor.py:153} INFO - Started process (PID=347) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:02:24.447+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:02:24.449+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:02:24.448+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:02:25.012+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:02:25.010+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:02:25.013+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:02:25.032+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.596 seconds
[2023-05-04T02:02:55.240+0000] {processor.py:153} INFO - Started process (PID=361) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:02:55.241+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:02:55.243+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:02:55.242+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:02:56.363+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:02:56.361+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:02:56.364+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:02:56.386+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.152 seconds
[2023-05-04T02:03:09.395+0000] {processor.py:153} INFO - Started process (PID=372) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:09.396+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:03:09.397+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:09.397+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:10.032+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:10.030+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import Git
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:03:10.033+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:10.055+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.668 seconds
[2023-05-04T02:03:11.543+0000] {processor.py:153} INFO - Started process (PID=376) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:11.544+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:03:11.545+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:11.545+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:12.235+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:12.233+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:03:12.235+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:12.255+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.719 seconds
[2023-05-04T02:03:23.101+0000] {processor.py:153} INFO - Started process (PID=382) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:23.103+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:03:23.104+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:23.104+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:23.109+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:23.108+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 172
    @
    ^
SyntaxError: invalid syntax
[2023-05-04T02:03:23.110+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:23.141+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.046 seconds
[2023-05-04T02:03:26.125+0000] {processor.py:153} INFO - Started process (PID=383) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:26.126+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:03:26.129+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:26.128+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:26.793+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:26.792+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:03:26.794+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:26.814+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.695 seconds
[2023-05-04T02:03:47.039+0000] {processor.py:153} INFO - Started process (PID=395) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:47.041+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:03:47.043+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:47.043+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:47.049+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:47.048+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 162
    with TaskGroup(group_id='group1') as gr1:
       ^
IndentationError: expected an indented block
[2023-05-04T02:03:47.050+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:47.087+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.058 seconds
[2023-05-04T02:03:52.071+0000] {processor.py:153} INFO - Started process (PID=396) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:52.072+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:03:52.073+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:52.072+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:52.076+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:52.075+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 161
    clone_repo=
              ^
SyntaxError: invalid syntax
[2023-05-04T02:03:52.077+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:52.100+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.034 seconds
[2023-05-04T02:03:57.180+0000] {processor.py:153} INFO - Started process (PID=397) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:57.183+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:03:57.186+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:57.185+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:57.884+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:57.881+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:03:57.885+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:57.922+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.766 seconds
[2023-05-04T02:03:59.972+0000] {processor.py:153} INFO - Started process (PID=401) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:03:59.973+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:03:59.974+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:03:59.974+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:00.634+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:04:00.632+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:04:00.635+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:00.657+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.688 seconds
[2023-05-04T02:04:14.770+0000] {processor.py:153} INFO - Started process (PID=414) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:14.771+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:04:14.772+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:04:14.772+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:15.382+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:04:15.381+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:04:15.383+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:15.402+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.636 seconds
[2023-05-04T02:04:20.484+0000] {processor.py:153} INFO - Started process (PID=418) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:20.486+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:04:20.488+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:04:20.488+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:21.159+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:04:21.158+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:04:21.160+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:21.178+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.702 seconds
[2023-05-04T02:04:22.116+0000] {processor.py:153} INFO - Started process (PID=422) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:22.117+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:04:22.118+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:04:22.118+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:22.707+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:04:22.705+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:04:22.707+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:22.726+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.614 seconds
[2023-05-04T02:04:29.296+0000] {processor.py:153} INFO - Started process (PID=426) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:29.306+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:04:29.308+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:04:29.307+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:29.952+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:04:29.950+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:04:29.952+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:29.973+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.691 seconds
[2023-05-04T02:04:33.826+0000] {processor.py:153} INFO - Started process (PID=430) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:33.826+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:04:33.827+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:04:33.827+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:34.400+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:04:34.398+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:04:34.400+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:04:34.426+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.604 seconds
[2023-05-04T02:05:04.744+0000] {processor.py:153} INFO - Started process (PID=443) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:05:04.749+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:05:04.751+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:05:04.751+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:05:05.343+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:05:05.341+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:05:05.343+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:05:05.361+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.631 seconds
[2023-05-04T02:05:35.654+0000] {processor.py:153} INFO - Started process (PID=456) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:05:35.656+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:05:35.658+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:05:35.657+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:05:36.373+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:05:36.372+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:05:36.373+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:05:36.392+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.744 seconds
[2023-05-04T02:06:06.700+0000] {processor.py:153} INFO - Started process (PID=477) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:06:06.702+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:06:06.704+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:06:06.703+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:06:07.278+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:06:07.276+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:06:07.278+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:06:07.298+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.613 seconds
[2023-05-04T02:06:37.575+0000] {processor.py:153} INFO - Started process (PID=490) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:06:37.576+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:06:37.579+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:06:37.578+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:06:38.339+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:06:38.337+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:06:38.339+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:06:38.361+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.795 seconds
[2023-05-04T02:07:08.649+0000] {processor.py:153} INFO - Started process (PID=503) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:07:08.651+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:07:08.652+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:07:08.652+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:07:09.199+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:07:09.198+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:07:09.199+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:07:09.219+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.576 seconds
[2023-05-04T02:07:39.515+0000] {processor.py:153} INFO - Started process (PID=516) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:07:39.516+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:07:39.517+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:07:39.517+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:07:40.088+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:07:40.087+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:07:40.089+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:07:40.108+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.599 seconds
[2023-05-04T02:07:51.759+0000] {processor.py:153} INFO - Started process (PID=522) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:07:51.762+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:07:51.764+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:07:51.763+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:07:52.672+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:07:52.671+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:07:52.673+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:07:52.694+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.949 seconds
[2023-05-04T02:08:06.175+0000] {processor.py:153} INFO - Started process (PID=526) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:06.176+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:08:06.177+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:06.177+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:06.930+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:06.928+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:08:06.930+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:06.952+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.784 seconds
[2023-05-04T02:08:11.874+0000] {processor.py:153} INFO - Started process (PID=537) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:11.877+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:08:11.879+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:11.878+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:12.639+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:12.637+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:08:12.639+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:12.661+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.799 seconds
[2023-05-04T02:08:16.040+0000] {processor.py:153} INFO - Started process (PID=541) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:16.041+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:08:16.043+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:16.042+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:16.903+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:16.893+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:08:16.903+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:16.985+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.951 seconds
[2023-05-04T02:08:25.782+0000] {processor.py:153} INFO - Started process (PID=547) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:25.784+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:08:25.785+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:25.785+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:25.789+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:25.788+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo=''
       ^
SyntaxError: invalid syntax
[2023-05-04T02:08:25.789+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:25.814+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.038 seconds
[2023-05-04T02:08:26.868+0000] {processor.py:153} INFO - Started process (PID=548) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:26.871+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:08:26.882+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:26.881+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:26.909+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:26.900+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo='pip install apache-airflow-providers-github'
       ^
SyntaxError: invalid syntax
[2023-05-04T02:08:26.910+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:26.956+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.117 seconds
[2023-05-04T02:08:27.892+0000] {processor.py:153} INFO - Started process (PID=549) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:27.893+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:08:27.894+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:27.894+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:27.899+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:27.897+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo=''
       ^
SyntaxError: invalid syntax
[2023-05-04T02:08:27.900+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:27.934+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.048 seconds
[2023-05-04T02:08:38.034+0000] {processor.py:153} INFO - Started process (PID=557) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:38.036+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:08:38.038+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:38.038+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:38.045+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:38.043+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo='data-engineering-zoomcamp',
       ^
SyntaxError: invalid syntax
[2023-05-04T02:08:38.046+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:38.078+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.056 seconds
[2023-05-04T02:08:44.113+0000] {processor.py:153} INFO - Started process (PID=558) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:44.114+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:08:44.117+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:44.117+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:44.122+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:44.120+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo='data-engineering-zoomcamp',
       ^
SyntaxError: invalid syntax
[2023-05-04T02:08:44.122+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:44.155+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.048 seconds
[2023-05-04T02:08:46.180+0000] {processor.py:153} INFO - Started process (PID=559) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:46.181+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:08:46.183+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:46.182+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:46.189+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:46.188+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo='data-engineering-zoomcamp',
       ^
SyntaxError: invalid syntax
[2023-05-04T02:08:46.190+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:46.225+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.051 seconds
[2023-05-04T02:08:49.218+0000] {processor.py:153} INFO - Started process (PID=560) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:49.219+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:08:49.220+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:49.220+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:49.226+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:49.224+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo='data-engineering-zoomcamp',
       ^
SyntaxError: invalid syntax
[2023-05-04T02:08:49.226+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:49.260+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.048 seconds
[2023-05-04T02:08:54.127+0000] {processor.py:153} INFO - Started process (PID=563) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:54.127+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:08:54.128+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:54.128+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:54.131+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:08:54.130+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo='data-engineering-zoomcamp',
       ^
SyntaxError: invalid syntax
[2023-05-04T02:08:54.132+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:08:54.155+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.032 seconds
[2023-05-04T02:09:00.226+0000] {processor.py:153} INFO - Started process (PID=564) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:00.231+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:09:00.234+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:00.233+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:00.242+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:00.240+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo='data-engineering-zoomcamp',
       ^
SyntaxError: invalid syntax
[2023-05-04T02:09:00.242+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:00.272+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.063 seconds
[2023-05-04T02:09:05.280+0000] {processor.py:153} INFO - Started process (PID=565) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:05.281+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:09:05.281+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:05.281+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:05.285+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:05.284+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo='data-engineering-zoomcamp',
       ^
SyntaxError: invalid syntax
[2023-05-04T02:09:05.286+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:05.312+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.038 seconds
[2023-05-04T02:09:11.423+0000] {processor.py:153} INFO - Started process (PID=572) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:11.428+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:09:11.434+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:11.432+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:11.454+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:11.450+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo='data-engineering-zoomcamp',
       ^
SyntaxError: invalid syntax
[2023-05-04T02:09:11.456+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:11.510+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.116 seconds
[2023-05-04T02:09:12.550+0000] {processor.py:153} INFO - Started process (PID=573) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:12.555+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:09:12.557+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:12.556+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:12.564+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:12.562+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo='data-engineering-zoomcamp',
       ^
SyntaxError: invalid syntax
[2023-05-04T02:09:12.565+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:12.600+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.065 seconds
[2023-05-04T02:09:42.846+0000] {processor.py:153} INFO - Started process (PID=583) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:42.850+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:09:42.852+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:42.851+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:42.858+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:42.855+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo='data-engineering-zoomcamp',
       ^
SyntaxError: invalid syntax
[2023-05-04T02:09:42.859+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:42.892+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.061 seconds
[2023-05-04T02:09:47.933+0000] {processor.py:153} INFO - Started process (PID=584) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:47.937+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:09:47.939+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:47.939+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:47.947+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:47.945+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 724, in exec_module
  File "<frozen importlib._bootstrap_external>", line 860, in get_code
  File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 163
    repo='data-engineering-zoomcamp',
       ^
SyntaxError: invalid syntax
[2023-05-04T02:09:47.948+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:47.978+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.070 seconds
[2023-05-04T02:09:50.051+0000] {processor.py:153} INFO - Started process (PID=585) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:50.054+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:09:50.056+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:50.055+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:50.671+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:09:50.670+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:09:50.672+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:09:50.695+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.673 seconds
[2023-05-04T02:10:10.227+0000] {processor.py:153} INFO - Started process (PID=598) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:10:10.228+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:10:10.229+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:10:10.229+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:10:10.970+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:10:10.968+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:10:10.970+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:10:10.997+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.773 seconds
[2023-05-04T02:10:14.101+0000] {processor.py:153} INFO - Started process (PID=602) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:10:14.102+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:10:14.104+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:10:14.103+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:10:14.746+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:10:14.744+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:10:14.746+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:10:14.767+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.680 seconds
[2023-05-04T02:10:45.064+0000] {processor.py:153} INFO - Started process (PID=614) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:10:45.068+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:10:45.069+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:10:45.068+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:10:45.978+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:10:45.976+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:10:45.979+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:10:46.006+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.947 seconds
[2023-05-04T02:11:16.376+0000] {processor.py:153} INFO - Started process (PID=628) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:11:16.379+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:11:16.381+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:11:16.380+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:11:17.244+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:11:17.242+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:11:17.245+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:11:17.266+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.901 seconds
[2023-05-04T02:11:47.600+0000] {processor.py:153} INFO - Started process (PID=641) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:11:47.604+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:11:47.606+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:11:47.606+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:11:48.251+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:11:48.249+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:11:48.251+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:11:48.275+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.691 seconds
[2023-05-04T02:12:18.601+0000] {processor.py:153} INFO - Started process (PID=654) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:12:18.603+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:12:18.604+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:12:18.604+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:12:19.268+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:12:19.266+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:12:19.268+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:12:19.305+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.717 seconds
[2023-05-04T02:12:49.636+0000] {processor.py:153} INFO - Started process (PID=667) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:12:49.638+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:12:49.639+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:12:49.639+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:12:50.208+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:12:50.207+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:12:50.209+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:12:50.228+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.604 seconds
[2023-05-04T02:13:20.511+0000] {processor.py:153} INFO - Started process (PID=679) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:13:20.513+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:13:20.514+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:13:20.514+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:13:21.122+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:13:21.121+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:13:21.122+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:13:21.142+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.636 seconds
[2023-05-04T02:13:51.422+0000] {processor.py:153} INFO - Started process (PID=691) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:13:51.424+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:13:51.426+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:13:51.425+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:13:52.164+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:13:52.163+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:13:52.165+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:13:52.186+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.769 seconds
[2023-05-04T02:14:22.453+0000] {processor.py:153} INFO - Started process (PID=704) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:14:22.455+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:14:22.456+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:14:22.456+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:14:23.454+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:14:23.444+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:14:23.454+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:14:23.533+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.086 seconds
[2023-05-04T02:14:53.822+0000] {processor.py:153} INFO - Started process (PID=717) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:14:53.825+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:14:53.827+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:14:53.827+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:14:54.569+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:14:54.567+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:14:54.569+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:14:54.596+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.784 seconds
[2023-05-04T02:15:24.736+0000] {processor.py:153} INFO - Started process (PID=730) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:15:24.738+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:15:24.739+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:15:24.739+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:15:25.326+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:15:25.325+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:15:25.327+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:15:25.346+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.616 seconds
[2023-05-04T02:15:55.646+0000] {processor.py:153} INFO - Started process (PID=745) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:15:55.647+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:15:55.648+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:15:55.648+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:15:56.212+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:15:56.211+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:15:56.213+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:15:56.236+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.597 seconds
[2023-05-04T02:16:26.602+0000] {processor.py:153} INFO - Started process (PID=759) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:16:26.603+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:16:26.604+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:16:26.604+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:16:27.349+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:16:27.348+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:16:27.349+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:16:27.372+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.776 seconds
[2023-05-04T02:16:57.596+0000] {processor.py:153} INFO - Started process (PID=778) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:16:57.597+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:16:57.598+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:16:57.597+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:16:58.197+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:16:58.196+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:16:58.198+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:16:58.219+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.628 seconds
[2023-05-04T02:17:28.509+0000] {processor.py:153} INFO - Started process (PID=793) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:17:28.512+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:17:28.515+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:17:28.515+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:17:29.288+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:17:29.286+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:17:29.289+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:17:29.311+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.827 seconds
[2023-05-04T02:17:32.382+0000] {processor.py:153} INFO - Started process (PID=797) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:17:32.385+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:17:32.389+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:17:32.389+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:17:33.130+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:17:33.129+0000] {dagbag.py:341} ERROR - Failed to import: /opt/airflow/dags/etl_web_to_s3.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/dagbag.py", line 337, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/etl_web_to_s3.py", line 21, in <module>
    from airflow.providers.github.operators.github import GithubOperator
ModuleNotFoundError: No module named 'airflow.providers.github'
[2023-05-04T02:17:33.131+0000] {processor.py:755} WARNING - No viable dags retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:17:33.151+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.781 seconds
[2023-05-04T02:17:47.284+0000] {processor.py:153} INFO - Started process (PID=810) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:17:47.285+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:17:47.286+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:17:47.286+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:17:48.210+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:17:48.363+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:17:48.362+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:17:48.426+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:17:48.426+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:17:48.464+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.186 seconds
[2023-05-04T02:18:18.655+0000] {processor.py:153} INFO - Started process (PID=823) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:18:18.657+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:18:18.659+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:18:18.659+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:18:19.532+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:18:19.553+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:18:19.553+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:18:19.586+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:18:19.585+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:18:19.607+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.957 seconds
[2023-05-04T02:18:49.718+0000] {processor.py:153} INFO - Started process (PID=838) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:18:49.719+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:18:49.720+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:18:49.720+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:18:50.556+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:18:50.583+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:18:50.583+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:18:50.617+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:18:50.617+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:18:50.642+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.929 seconds
[2023-05-04T02:19:21.037+0000] {processor.py:153} INFO - Started process (PID=852) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:19:21.038+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:19:21.040+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:19:21.039+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:19:21.867+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:19:21.888+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:19:21.887+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:19:21.920+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:19:21.920+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:19:21.952+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.920 seconds
[2023-05-04T02:19:52.650+0000] {processor.py:153} INFO - Started process (PID=866) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:19:52.651+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:19:52.654+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:19:52.654+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:19:53.391+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:19:53.411+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:19:53.411+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:19:53.446+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:19:53.446+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:19:53.473+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.833 seconds
[2023-05-04T02:20:23.893+0000] {processor.py:153} INFO - Started process (PID=880) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:20:23.896+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:20:23.897+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:20:23.897+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:20:24.579+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:20:24.602+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:20:24.601+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:20:24.642+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:20:24.642+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:20:24.669+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.786 seconds
[2023-05-04T02:20:54.956+0000] {processor.py:153} INFO - Started process (PID=894) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:20:54.957+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:20:54.957+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:20:54.957+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:20:55.875+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:20:55.903+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:20:55.903+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:20:55.945+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:20:55.944+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:20:55.974+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.023 seconds
[2023-05-04T02:21:26.176+0000] {processor.py:153} INFO - Started process (PID=908) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:21:26.178+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:21:26.181+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:21:26.180+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:21:26.917+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:21:26.957+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:21:26.956+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:21:27.004+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:21:27.004+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:21:27.040+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.888 seconds
[2023-05-04T02:21:57.548+0000] {processor.py:153} INFO - Started process (PID=922) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:21:57.552+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:21:57.554+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:21:57.553+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:21:58.233+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:21:58.254+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:21:58.254+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:21:58.290+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:21:58.290+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:21:58.316+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.783 seconds
[2023-05-04T02:22:28.837+0000] {processor.py:153} INFO - Started process (PID=936) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:22:28.841+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:22:28.844+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:22:28.843+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:22:29.536+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:22:29.557+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:22:29.557+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:22:29.592+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:22:29.592+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:22:29.619+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.799 seconds
[2023-05-04T02:23:00.308+0000] {processor.py:153} INFO - Started process (PID=950) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:23:00.314+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:23:00.320+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:23:00.319+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:23:01.016+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:23:01.036+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:23:01.035+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:23:01.072+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:23:01.072+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:23:01.105+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.822 seconds
[2023-05-04T02:23:31.674+0000] {processor.py:153} INFO - Started process (PID=964) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:23:31.678+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:23:31.679+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:23:31.679+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:23:32.366+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:23:32.393+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:23:32.392+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:23:32.426+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:23:32.426+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:23:32.449+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.787 seconds
[2023-05-04T02:24:03.019+0000] {processor.py:153} INFO - Started process (PID=978) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:24:03.020+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:24:03.021+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:24:03.021+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:24:03.785+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:24:03.807+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:24:03.806+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:24:03.840+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:24:03.840+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:24:03.865+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.852 seconds
[2023-05-04T02:24:34.477+0000] {processor.py:153} INFO - Started process (PID=992) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:24:34.484+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:24:34.489+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:24:34.487+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:24:35.225+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:24:35.247+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:24:35.247+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:24:35.280+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:24:35.279+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:24:35.315+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.865 seconds
[2023-05-04T02:25:05.854+0000] {processor.py:153} INFO - Started process (PID=1006) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:25:05.860+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:25:05.865+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:25:05.864+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:25:06.563+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:25:06.583+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:25:06.582+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:25:06.620+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:25:06.620+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:25:06.646+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.815 seconds
[2023-05-04T02:25:37.362+0000] {processor.py:153} INFO - Started process (PID=1020) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:25:37.365+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:25:37.366+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:25:37.366+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:25:38.027+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:25:38.047+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:25:38.047+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:25:38.091+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:25:38.091+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:25:38.117+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.764 seconds
[2023-05-04T02:26:08.831+0000] {processor.py:153} INFO - Started process (PID=1034) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:26:08.833+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:26:08.835+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:26:08.834+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:26:09.497+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:26:09.518+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:26:09.517+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:26:09.553+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:26:09.552+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:26:09.576+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.751 seconds
[2023-05-04T02:26:39.979+0000] {processor.py:153} INFO - Started process (PID=1048) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:26:39.981+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:26:39.983+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:26:39.982+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:26:40.800+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:26:40.825+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:26:40.824+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:26:40.862+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:26:40.862+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:26:40.886+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.914 seconds
[2023-05-04T02:27:11.780+0000] {processor.py:153} INFO - Started process (PID=1062) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:27:11.782+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:27:11.784+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:27:11.784+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:27:12.543+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:27:12.567+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:27:12.566+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:27:12.606+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:27:12.606+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:27:12.632+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.856 seconds
[2023-05-04T02:27:42.734+0000] {processor.py:153} INFO - Started process (PID=1076) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:27:42.735+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:27:42.736+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:27:42.736+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:27:43.699+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:27:43.721+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:27:43.721+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:27:43.757+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:27:43.756+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:27:43.780+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.054 seconds
[2023-05-04T02:28:14.132+0000] {processor.py:153} INFO - Started process (PID=1089) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:28:14.134+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:28:14.135+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:28:14.134+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:28:14.887+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:28:14.907+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:28:14.906+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:28:14.938+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:28:14.937+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:28:14.959+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.832 seconds
[2023-05-04T02:28:45.018+0000] {processor.py:153} INFO - Started process (PID=1109) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:28:45.021+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:28:45.022+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:28:45.021+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:28:45.733+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:28:45.763+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:28:45.762+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:28:45.809+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:28:45.808+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:28:45.840+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.828 seconds
[2023-05-04T02:29:16.151+0000] {processor.py:153} INFO - Started process (PID=1123) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:29:16.152+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:29:16.153+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:29:16.152+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:29:16.732+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:29:16.751+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:29:16.750+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:29:16.782+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:29:16.782+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:29:16.804+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.657 seconds
[2023-05-04T02:29:47.160+0000] {processor.py:153} INFO - Started process (PID=1137) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:29:47.164+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:29:47.165+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:29:47.165+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:29:47.868+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:29:47.892+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:29:47.891+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:29:47.927+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:29:47.927+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:29:47.950+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.800 seconds
[2023-05-04T02:30:18.282+0000] {processor.py:153} INFO - Started process (PID=1153) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:30:18.283+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:30:18.283+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:30:18.283+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:30:18.879+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:30:18.897+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:30:18.897+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:30:18.929+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:30:18.929+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:30:18.951+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.675 seconds
[2023-05-04T02:30:49.541+0000] {processor.py:153} INFO - Started process (PID=1166) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:30:49.545+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:30:49.547+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:30:49.546+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:30:50.574+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:30:50.595+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:30:50.594+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:30:50.650+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:30:50.650+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:30:50.685+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.152 seconds
[2023-05-04T02:31:20.940+0000] {processor.py:153} INFO - Started process (PID=1179) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:31:20.941+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:31:20.942+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:31:20.942+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:31:21.630+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:31:21.655+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:31:21.654+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:31:21.698+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:31:21.698+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:31:21.723+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.788 seconds
[2023-05-04T02:31:52.045+0000] {processor.py:153} INFO - Started process (PID=1192) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:31:52.047+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:31:52.048+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:31:52.048+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:31:52.709+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:31:52.728+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:31:52.727+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:31:52.757+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:31:52.757+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:31:52.779+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.743 seconds
[2023-05-04T02:32:23.110+0000] {processor.py:153} INFO - Started process (PID=1206) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:32:23.115+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:32:23.120+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:32:23.119+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:32:23.946+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:32:23.972+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:32:23.971+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:32:24.018+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:32:24.017+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:32:24.047+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.957 seconds
[2023-05-04T02:32:54.359+0000] {processor.py:153} INFO - Started process (PID=1219) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:32:54.363+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:32:54.365+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:32:54.364+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:32:54.985+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:32:55.004+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:32:55.003+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:32:55.036+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:32:55.036+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:32:55.058+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.712 seconds
[2023-05-04T02:33:25.370+0000] {processor.py:153} INFO - Started process (PID=1232) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:33:25.371+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:33:25.373+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:33:25.372+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:33:26.200+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:33:26.221+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:33:26.221+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:33:26.254+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:33:26.254+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:33:26.275+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.910 seconds
[2023-05-04T02:33:56.625+0000] {processor.py:153} INFO - Started process (PID=1245) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:33:56.627+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:33:56.628+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:33:56.628+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:33:57.342+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:33:57.374+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:33:57.373+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:33:57.417+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:33:57.416+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:33:57.444+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.826 seconds
[2023-05-04T02:34:27.909+0000] {processor.py:153} INFO - Started process (PID=1258) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:34:27.910+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:34:27.911+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:34:27.911+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:34:29.822+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:34:29.908+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:34:29.907+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:34:29.986+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:34:29.986+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:34:30.061+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 2.182 seconds
[2023-05-04T02:35:00.247+0000] {processor.py:153} INFO - Started process (PID=1271) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:35:00.251+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:35:00.253+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:35:00.252+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:35:01.728+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:35:01.779+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:35:01.779+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:35:01.902+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:35:01.902+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:35:01.974+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.734 seconds
[2023-05-04T02:35:32.584+0000] {processor.py:153} INFO - Started process (PID=1284) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:35:32.587+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:35:32.588+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:35:32.588+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:35:33.759+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:35:33.803+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:35:33.802+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:35:33.868+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:35:33.868+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:35:33.904+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.326 seconds
[2023-05-04T02:36:04.113+0000] {processor.py:153} INFO - Started process (PID=1298) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:36:04.114+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:36:04.115+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:36:04.115+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:36:05.188+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:36:05.235+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:36:05.234+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:36:05.316+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:36:05.316+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:36:05.348+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.243 seconds
[2023-05-04T02:36:35.578+0000] {processor.py:153} INFO - Started process (PID=1311) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:36:35.580+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:36:35.582+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:36:35.581+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:36:36.551+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:36:36.583+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:36:36.583+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:36:36.648+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:36:36.647+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:36:36.687+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.116 seconds
[2023-05-04T02:37:06.987+0000] {processor.py:153} INFO - Started process (PID=1324) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:37:06.988+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:37:06.989+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:37:06.989+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:37:07.844+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:37:07.872+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:37:07.871+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:37:07.908+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:37:07.908+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:37:07.932+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.950 seconds
[2023-05-04T02:37:38.395+0000] {processor.py:153} INFO - Started process (PID=1336) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:37:38.397+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:37:38.398+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:37:38.398+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:37:39.227+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:37:39.248+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:37:39.247+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:37:39.288+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:37:39.288+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:37:39.312+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.921 seconds
[2023-05-04T02:38:09.561+0000] {processor.py:153} INFO - Started process (PID=1348) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:38:09.564+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:38:09.568+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:38:09.567+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:38:10.252+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:38:10.278+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:38:10.277+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:38:10.315+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:38:10.315+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:38:10.340+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.795 seconds
[2023-05-04T02:38:40.574+0000] {processor.py:153} INFO - Started process (PID=1361) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:38:40.575+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:38:40.576+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:38:40.575+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:38:41.333+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:38:41.365+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:38:41.364+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:38:41.410+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:38:41.410+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:38:41.448+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.878 seconds
[2023-05-04T02:39:11.680+0000] {processor.py:153} INFO - Started process (PID=1374) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:39:11.681+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:39:11.682+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:39:11.682+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:39:12.767+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:39:12.806+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:39:12.804+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:39:12.857+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:39:12.856+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:39:12.888+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.218 seconds
[2023-05-04T02:39:43.203+0000] {processor.py:153} INFO - Started process (PID=1387) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:39:43.205+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:39:43.205+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:39:43.205+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:39:43.917+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:39:43.946+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:39:43.946+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:39:43.992+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:39:43.992+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:39:44.022+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.825 seconds
[2023-05-04T02:40:14.317+0000] {processor.py:153} INFO - Started process (PID=1399) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:40:14.318+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:40:14.319+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:40:14.319+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:40:15.282+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:40:15.322+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:40:15.322+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:40:15.376+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:40:15.376+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:40:15.403+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 1.092 seconds
[2023-05-04T02:40:45.560+0000] {processor.py:153} INFO - Started process (PID=1412) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:40:45.561+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:40:45.562+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:40:45.562+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:40:46.457+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:40:46.483+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:40:46.483+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:40:46.518+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:40:46.518+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:40:46.545+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.993 seconds
[2023-05-04T02:41:16.642+0000] {processor.py:153} INFO - Started process (PID=1424) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:41:16.644+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:41:16.645+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:41:16.645+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:41:17.272+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:41:17.293+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:41:17.293+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:41:17.326+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:41:17.326+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:41:17.351+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.717 seconds
[2023-05-04T02:41:47.559+0000] {processor.py:153} INFO - Started process (PID=1438) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:41:47.561+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:41:47.563+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:41:47.562+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:41:48.256+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:41:48.279+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:41:48.278+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:41:48.316+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:41:48.316+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:41:48.343+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.788 seconds
[2023-05-04T02:42:18.612+0000] {processor.py:153} INFO - Started process (PID=1451) to work on /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:42:18.615+0000] {processor.py:743} INFO - Processing file /opt/airflow/dags/etl_web_to_s3.py for tasks to queue
[2023-05-04T02:42:18.617+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:42:18.616+0000] {dagbag.py:532} INFO - Filling up the DagBag from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:42:19.212+0000] {processor.py:753} INFO - DAG(s) dict_keys(['etl_web_to_s3']) retrieved from /opt/airflow/dags/etl_web_to_s3.py
[2023-05-04T02:42:19.230+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:42:19.230+0000] {dag.py:2705} INFO - Sync 1 DAGs
[2023-05-04T02:42:19.292+0000] {logging_mixin.py:137} INFO - [2023-05-04T02:42:19.292+0000] {dag.py:3456} INFO - Setting next_dagrun for etl_web_to_s3 to 2023-04-26T00:00:00+00:00, run_after=2023-04-27T00:00:00+00:00
[2023-05-04T02:42:19.323+0000] {processor.py:175} INFO - Processing /opt/airflow/dags/etl_web_to_s3.py took 0.724 seconds
